openmeter/entitlement/driver/parser.go

```diff
@@ -54,7 +54,13 @@ func (parser) ToMetered(e *entitlement.Entitlement) (*api.EntitlementMetered, er
 		CurrentUsagePeriod:     *mapPeriod(e.CurrentUsagePeriod),
 		LastReset:              metered.LastReset,
 		PreserveOverageAtReset: convert.ToPointer(metered.PreserveOverageAtReset),
-	}, nil
+	}
+
+	if e.SubscriptionManaged {
+		(*apiRes.Metadata)["om_subscription_managed"] = "true"
```

Comment:
You could use a constant for the key.

Reply:
`"true"` isn't the best value as a string, does it make sense to do something like: `om_managed_by=subscription` or something? Please also check if the map is `nil`.

Reply:
We should probably also reserve `openmeter.` prefix not to be set by users.

Reply:
(E.g. notifications uses Annotations to signify that it's not an user servicable data).